livespark-common-widgets/livespark-crud-component/src/main/java/org/livespark/widgets/crud/client/resources/i18n/CrudComponentConstants.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.livespark.widgets.crud.client.resources.i18n;
+
+import org.jboss.errai.ui.shared.api.annotations.TranslationKey;
+
+public interface CrudComponentConstants {
+
+    @TranslationKey( defaultValue = "")
+    String CrudComponentViewImpl_newInstanceButton = "CrudComponentViewImpl.newInstanceButton";
```

Comment:
Using underscores in identifiers is deprecated in Java 8 (and will show warnings in the build). I see why you did it in this case, but I think it's best to remove them.